libs/reports-list/src/lib/report-modal/report-modal.component.html

```diff
@@ -0,0 +1,8 @@
+<h1 mat-dialog-title>דיווח על חיסונים</h1>
+<div mat-dialog-content>
+  <mat-label>קופה: {{healthCareService}}</mat-label><br>
```

Comment:
Consider having a separate html element for the heading part (e.g. "קופה:") and separate element for the value, so we can easily design it in CSS.
Something like:
<mat-label>
  <div class="labelHeading">קופה:</div>
  <div class="labelValue">{{healthCareService}}</mat-label>
</mat-label>